kedro-org/kedro-viz
# Fix for Kedro Viz Connection Error

## Description

Resolves #1277 

## Development notes

- Start Fast API server using multiprocessing similar to jupyter launcher with/without --autoreload option
-  Wait for server to start before opening web browser 

![Resolved_Connection_Error](https://github.com/kedro-org/kedro-viz/assets/87735534/6f87f319-8994-4b2d-ad18-ffef5f96ec6c)


## QA notes

- Checkout branch `fix/connection-error`
- Build frontend using `make build`
- Build backend using `pip install -e package`
- cd demo-project, execute command `kedro viz` or `kedro viz --autoreload`
- Browser window should open after FAST API server is started and the browser should not dispay a connection error

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1507"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
